### ISSUE TITLE
Controlnet Support

### ISSUE THREAD

**Issue by Oguzhanercan** [NVlabs/Sana#73]
Thanks for your work, are you gonna train Control Nets for this model?

**Comment by able2608** [comment]
Judging from the network architecture, controlnet support seems to be possible. However, most controlnets come from the community, not from the original author. Perhaps we need to wait a while before someone is able to make one.

**Comment by lawrence-cj** [comment]
ControlNet will be released soon. Maybe in a half of a month, I think?🚀

**Comment by xieenze** [comment]
Yes we trained an in-house ControlNet-SANA already (scribble-based control map), please wait for the release~ 😄

**Comment by CorradoF** [comment]
> ControlNet will be released soon. Maybe in a half of a month, I think?🚀

Can you please add the tile deblur control net in your release? Like [https://huggingface.co/xinsir/controlnet-union-sdxl-1.0]
It gives details to Images, for example you extract low quality video frames and you can reconstruct them in 4k effortlessly

**Comment by fengweipeng** [comment]
> ControlNet will be released soon. Maybe in a half of a month, I think?🚀

> Yes we trained an in-house ControlNet-SANA already (scribble-based control map), please wait for the release~ 😄

It's been a month now, and ControlNet still hasn't been released. What difficulties have you encountered? When will it be released approximately? Will the 2K version be released first?

**Comment by marianbasti** [comment]
https://github.com/NVlabs/Sana/pull/177 is working on this
